packages/atclient/include/atclient/version.h

```diff
@@ -1,6 +1,6 @@
 #ifndef ATCLIENT_VERSION_H
 #define ATCLIENT_VERSION_H
 
-#define ATCLIENT_ATSDK_VERSION "0.3.1"
+#define ATCLIENT_ATSDK_VERSION "0.3.0"
```

Comment:
Shouldn't it be 0.3.2?

Reply:
Corrected in [26239f0](https://github.com/atsign-foundation/at_c/pull/445/commits/26239f0c3fb4cd98bf401a70aaf60a01b80f23e6)

Reply:
Not sure why it was 0.3.0, I changed it from 0.3.1 to 0.3.2 but forgot to save the file

Reply:
I'm still seeing 0.3.0